routes/web.php

```diff
@@ -17,16 +17,16 @@
 });
 
 Route::prefix('api/')->group(function () {
```

Comment:
I know it's not part of the task but Laravel ships with an `api.php` file inside the `routes` directory. Might want to make a new task to add these routes there that way you don't have to adjust the `VerifyCsrfToken.php` exception.

Reply:
sure thing boss 🤖